### ISSUE TITLE
How to train If I add some pictures? 

### ISSUE THREAD

**Issue by mahxn0** [AlexeyAB/darknet#1209]
First, I trained a weights not detect well
Then, I add some pictures to train_img, can I  train it  after first trained model?   Is there some  defferent  train with  darknet53.conv?

```
After  ./darknet detector train cfg/coco.data cfg/yolov3.cfg darknet53.conv.... 6000 pictures , 40000 iters I get a model yolo_final.weights not  take good.  Then I add  500  negative pictures to train_img ,  can i fine tuning  from  first result  yolov3_final.weights?  not start from darknet53.conv ...
```

**Comment by AlexeyAB** [comment]
@mahxn0 Yes, you can continue training from `yolov3_40000.weights` instead of darknet53.conv, if there wasn't changed number of classes.

**Comment by mahxn0** [comment]
thank you, If continue,  train.list  is all  of my picture  include  500 negative pic?  how to  set iter num? like 45000? 50000?  or  you mean only used yolo40000.weights as pretrain model ?

**Comment by mahxn0** [comment]
thank you, If continue,  train.list  is all  of my picture  include  500 negative pic?  how to  set iter num? like 45000? 50000?  or  you mean only used yolo40000.weights as pretrain model ?

**Comment by AlexeyAB** [comment]
* `train.txt` should contain old+new images.
* Just run `./darknet detector train cfg/coco.data cfg/yolov3.cfg backup/yolov3_40000.weights`

You can increase `steps=50000,60000` and `max_batches=70000`
